api/db/changelog/db.changelog-6.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="blrubenstein" id="changelog-4">
+    <addColumn tableName="workspace">
+      <column name="rp_review_requested" type="boolean"/>
+      <column name="rp_approved" type="boolean">
+        <constraints nullable="true"/>
+      </column>
+      <column name="rp_time_requested" type="datetime">
```

Comment:
MySQL datetimes are all UTC / timezone agnostic, right?

Reply:
Quick googling says that is correct.